iceoryx_posh/include/iceoryx_posh/roudi/port_pool.hpp

```diff
@@ -64,27 +65,27 @@ class PortPool
 
     cxx::expected<PublisherPortRouDiType::MemberType_t*, PortPoolError>
     addPublisherPort(const capro::ServiceDescription& serviceDescription,
-                     const uint64_t& historyCapacity,
                      mepoo::MemoryManager* const memoryManager,
                      const ProcessName_t& applicationName,
+                     const popo::PublisherOptions& publisherOptions = popo::PublisherOptions(),
                      const mepoo::MemoryInfo& memoryInfo = mepoo::MemoryInfo()) noexcept;
 
     cxx::expected<SubscriberPortType::MemberType_t*, PortPoolError>
     addSubscriberPort(const capro::ServiceDescription& serviceDescription,
-                      const uint64_t& historyRequest,
                       const ProcessName_t& applicationName,
+                      const popo::SubscriberOptions& subscriberOptions,
```

Comment:
also here, why a default for the publisher but not for the subscriber?

Reply:
removed from publisher